lib/msal-common/src/client/SPAClient.ts

```diff
@@ -213,7 +213,13 @@ export class SPAClient extends BaseClient {
             // Get current cached tokens
             const cachedTokenItem = this.getCachedTokens(requestScopes, acquireTokenAuthority.canonicalAuthority, request.resource, account && account.homeAccountIdentifier);
             const expirationSec = Number(cachedTokenItem.value.expiresOnSec);
+            console.log(TimeUtils.nowSeconds());
```

Comment:
Can we remove all these console.log statements?

Reply:
done